src/Files.App/Services/App/AppThemeModeService.cs

```diff
@@ -61,6 +67,10 @@ public AppThemeModeService()
 
 			// Registering to color changes, so that we can notice when changed system theme mode
 			UISettings.ColorValuesChanged += UISettings_ColorValuesChanged;
+
+			var windowId = MainWindow.Instance.AppWindow.Id;
+			ThemeSettings = ThemeSettings.CreateForWindowId(windowId);
+			ThemeSettings.Changed += (s, e) => { IsHighContrastChanged?.Invoke(this, s.HighContrast); };
```

Comment:
Should this be split onto separate lines? I'm fine with either way but I wanted to point this out just in case.

Reply:
I’m fine either but I prefer this.